include/swift/Driver/Driver.h

```diff
@@ -112,6 +112,8 @@ class OutputInfo {
   /// The path to the SDK against which to build.
   /// (If empty, this implies no SDK.)
   std::string SDKPath;
+
+  enum SanitizerKind SelectedSanitizer;
```

Comment:
This will cause redeclaration errors since it has an underlying type in one declaration but not here. Please just include the header.